### ISSUE TITLE
[Accessibility] Workbench Page: Back button and Option dropdown not keyboard accessible

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#997]
**Describe the bug**
The back button and options dropdown on the Workbench page are not keyboard focusable.

**To Reproduce**
1. Go to 'Workbench' in the nav bar
2. Press the tab key on your keyboard
3. See error

**Expected behavior**
Keyboard users should be able to navigate to and interact with the back button and options dropdown
